src/syscall_handler.c

```diff
@@ -112,6 +112,79 @@ int sys_close( int fd )
 	return ENOSYS;
 }
 
+int sys_w_color( int wd, int r, int g, int b ) {
```

Comment:
Let's be bold and call this `draw_color`, `draw_rect` etc throughout.
Once we start abbreviating, things can go downhill quickly...